_includes/v23.1/sql/use-case-trigram-indexes.md

```diff
@@ -0,0 +1 @@
+Depending on your use case, you may prefer to use [trigram indexes](trigram-indexes.html) to do fuzzy string matching and pattern matching. For more information about use cases for trigram indexes that could make having full text search unnecessary, see the 2022 blog post [Use cases for trigram indexes (when not to use Full Text Search)](https://www.cockroachlabs.com/blog/use-cases-trigram-indexes/).
```

Comment:
Do you want to hyphenate "full text search" here? It seems like hyphenation is the way you've gone elsewhere? It's kind of a shame that the blog post title is not hyphenated